site/app/views/grading/ElectronicGraderView.php

```diff
@@ -135,32 +147,57 @@ public function statusPage(
                 $peer_percentage = 0;
                 $peer_total = 0;
                 $peer_graded = 0;
+                $entire_peer_total = 0;
+                $total_students_submitted = 0;
+                $total_grading_percentage = 0;
+                $entire_peer_graded = 0;
+                $entire_peer_percentage = 0;
+                if ($peer_count > 0 && array_key_exists("peer_stu_grad", $sections)) {
+                    if ($num_peer_components > 0) {
+                        $total_students_submitted =  floor(($sections['peer_stu_grad']['total_who_submitted']));
+                        $total_grading_percentage =  number_format(($graded_total / $total_students_submitted ) * 100, 1);
+                        $entire_peer_total =  floor(($sections['peer_stu_grad']['total_who_submitted']));
+                        $entire_peer_graded =  $sections['peer_stu_grad']['view_peer_graded_components'] / $num_peer_components;
+                    }
+                    else {
```

Comment:
This else statement is not needed, just leave the varaibles defined above at 0

Reply:
Recent commits update this,